yb_migrate/src/srcdb/mysql.go

```diff
@@ -72,6 +72,10 @@ func (ms *MySQL) GetAllTableNames() []string {
 	return tableNames
 }
 
+func (ms *MySQL) GetAllPartitionNames(tableName string) []string {
+	panic("Not Implemented")
```

Comment:
@amit-yb is it okay to panic() as it shows the complete stack trace? or we should use utils.ErrExit() here

Reply:
In its current form this function should never be called. If it gets called for PG or MySQL, we will get this panic message right away during testing. So, users will never see this panic.

Reply:
got it, thanks!